impl/src/main/java/com/okta/sdk/impl/oauth2/OAuth2AccessToken.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2020 Okta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.okta.sdk.impl.oauth2;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.time.Duration;
+import java.time.Instant;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class OAuth2AccessToken {
```

Comment:
This is fine for this resource, but you likely noticed that other resources in this SDK are structured differently, we can talk about that when we chat about the OpenAPI stuff.  (and it doesn't need to hold up this PR)

Reply:
sounds good.